.github/actions/put-contents/action.yml

```diff
@@ -0,0 +1,15 @@
+name: 'Push files'
```

Comment:
[nit] can we rename the directory `put-contents` to something similar?

Reply:
push-files